ledger_device_sdk/src/io.rs

```diff
@@ -129,15 +136,36 @@ pub struct ApduHeader {
 }
 
 impl Comm {
+    /// Creates a new [`Comm`] instance, which accepts any CLA APDU by default.
     pub const fn new() -> Self {
         Self {
             apdu_buffer: [0u8; 260],
             rx: 0,
             tx: 0,
             buttons: ButtonsState::new(),
+            expected_cla: None,
         }
     }
 
+    /// Defines [`Comm::expected_cla`] in order to reply automatically [`StatusWords::BadCla`] when
+    /// an incoming APDU has a CLA byte different from the given value.
+    ///
+    /// # Arguments
+    ///
+    /// * `cla` - Expected value for APDUs CLA byte.
+    ///
+    /// # Examples
+    ///
+    /// This method can be used when building an instance of [`Comm`]:
+    ///
+    /// ```
+    /// let mut comm = Comm::new().expect_cla(0xe0);
+    /// ```
+    pub fn expect_cla(mut self, cla: u8) -> Self {
```

Comment:
I would rather call this function `set_expected_cla`

Reply:
Good suggestion. Done.